src/components/stateless/specific/pages/CreateOfferPage/OriginSelector.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Label from '../../../generic/Label/Label';
+import SelectorField from '../../../generic/SelectorField/SelectorField';
+import styles from './CreateOfferPage.scss';
+
+export default class OriginSelector extends Component {
```

Comment:
OriginSelector, CategorySelector and QualitySelector do basicly the same. Maybe we should create something like CreateOfferSelector with props to label and possible values?